README.md

```diff
@@ -66,15 +66,71 @@ https://github.com/btcsuite/btcwallet/releases
 
 ### Windows/Linux/BSD/POSIX - Build from source
 
-- If necessary, install Go according to the installation instructions
-  here: http://golang.org/doc/install.  It is recommended to add
-  `$GOPATH/bin` to your `PATH` at this point.
+Building or updating from source requires the following build dependencies:
+
+- **Go 1.5 or 1.6**
+
+  Installation instructions can be found here: http://golang.org/doc/install.
+  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
+
+- **Glide**
+
+  Glide is used to manage project dependencies and provide reproducable builds.
+  To install:
+
+  `go get -u github.com/Masterminds/glide`
+
+Unfortunately, the use of `glide` prevents a handy tool such as `go get` from
+automatically downloading, building, and installing the source in a single
+command.  Instead, the latest project and dependency sources must be first
+obtained manually with `git` and `glide`, and then `go` is used to build and
+install the project.
+
+**Getting the source**:
+
+For a first time installation, the project and dependency sources can be
+obtained manually with `git` and `glide` (create directories as needed):
+
+```
+git clone https://github.com/btcsuite/btcwallet $GOPATH/src/github.com/btcsuite/btcwallet
+cd $GOPATH/src/github.com/btcsuite/btcwallet
+glide install
```

Comment:
This is confusing in my opinion.  The first thing I expect to see is a "quick start" that is the most common case which is getting the source (including all deps), building, and installing `btcwallet` _and_ all of its utilities.  As it standing, the user will have to do this and then know which section to choose.  That is not very user friendly.